src/components/filter-controls/AnchorHeader.tsx

```
@@ -0,0 +1,95 @@
+  onClick
+}: {
+  label: string;
+  icon: Icon;
+  color: string;
+  onRemove: () => void;
+  onClick: () => void;
+}) => (
+  <div
+    className="flex items-center gap-2 px-3 py-1.5 rounded-full bg-white/10 dark:bg-black/40 border border-white/20 shadow-sm transition-all hover:bg-white/20 cursor-pointer"
+    onClick={onClick}
+  >
```

Biome (2.4.4)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)